src/transaction.ts

```diff
@@ -1288,6 +1325,25 @@ export class Snapshot extends EventEmitter {
     return {params, paramTypes};
   }
 
+  /**
+   * Get directed read options
+   * @private
+   * @param {google.spanner.v1.IDirectedReadOptions} directedReadOptions Request directedReadOptions object.
+   */
+  protected _getDirectedReadOptions(
+    directedReadOptions: IDirectedReadOptions | null | undefined
```

Comment:
nit: In the interfaces we have mentioned `directedReadOptions?: IDirectedReadOptions | null `.
1. Here we are allowing `undefined` as well. In which case does this undefined get passed?
2. Also what would be the default value of `IDirectedReadOptions` when it is not set by user.